actionpack/lib/action_dispatch/middleware/static.rb

```diff
@@ -13,15 +13,13 @@ module ActionDispatch
   # located at `public/assets/application.js` if the file exists. If the file
   # does not exist, a 404 "File not Found" response will be returned.
   class FileHandler
-    def initialize(root, cache_control, index = 'index')
+    def initialize(root, index = 'index', headers: )
```

Comment:
I did a little investigation but I'm not sure if we should treat `ActionDispatch::FileHandler` as a public class. It's basically an inner class. Does it still have to be public?